README.md

```diff
@@ -18,7 +18,7 @@ fixes](https://github.com/git-for-windows/git/wiki/How-to-participate).
 Git - fast, scalable, distributed revision control system
 =========================================================
 
-Git is a fast, scalable, distributed revision control system with an
+Git is a fast, scalable, distributed revision/version control system with an
```

Comment:
Not sure about that... in any case, this touches a part of the README that is not specific to Windows, and should therefore be sent to the Git mailing list (you can open a PR at https://github.com/gitgitgadget/git to do so).